core/templates/domain/feedback_message/FeedbackMessageSummaryObjectFactory.ts

```diff
@@ -61,22 +61,19 @@ export class FeedbackMessageSummaryObjectFactory {
       newMessageId, newMessageText, null, null, null, null, authorUsername,
       authorPictureDataUrl, createdOnMsecs);
   }
-  // TODO(#7176): Replace 'any' with the exact type. This has been kept as
```

Comment:
I think this comment needs to stay (or else you should resolve any to the correct type in line 65).

Reply:
@nithusha21 ,I will undo that

Reply:
@nithusha21 I have done placing the comments as it was earlier.